src/extensions/filter-control/utils.js

```diff
@@ -73,56 +93,54 @@ export function sortSelectControl (selectControl, orderBy) {
   $selectControl.append($opts)
 }
 
-export function fixHeaderCSS ({ $tableHeader }) {
-  $tableHeader.css('height', '89px')
+export function fixHeaderCSS ({ $tableHeader }, pixels = '89px') {
+  if ($tableHeader.hasClass('table-sm') && pixels === '89px') {
```

Comment:
`table-sm` is a class of bootstrap so users of other css frameworks (or without) don't have that class.

Reply:
You're right. Let me think how I can improve this

Reply:
Still pending